plugins/org.eclipse.elk.alg.topdownpacking/build.properties

```diff
@@ -0,0 +1,5 @@
+source.. = src/
+output.. = bin/
+bin.includes = plugin.xml,\
```

Comment:
Why do you need a plugin.xml?

Reply:
I don't know, I think it was generated automatically